src/encoded/static/components/layout.js

```diff
@@ -54,13 +54,12 @@ var BlockEditModal = React.createClass({
 
     render: function() {
         var blocktype = globals.blocks.lookup(this.props.value);
-        var schema = blocktype.schema();
         var BlockEdit = blocktype.edit || FallbackBlockEdit;
         return (
-            <Modal actuator={this.props.actuator}>
+            <Modal ref="modal" actuator={this.props.actuator}>
```

Comment:
* I’m not clear on the reason for adding the `ref` to BlockEditModal and Modal, and then rendering when the component mounts. I’m wondering if this reveals a weakness in the Modal component itself (or possibly my out-of-date comments in modal.js)?